python-client/giskard/ml_worker/core/suite.py

```diff
@@ -224,6 +224,12 @@ def add_test(self, test_fn: Test, test_name: Optional[Union[int, str]] = None, *
         :return: The current instance of the test suite to allow chained calls.
         :rtype: TestSuite
         """
+        if isinstance(test_fn, bool) or isinstance(test_fn, TestResult):
+            raise TypeError("""
+            It seems that you executed your test inside of the 'add_test' method.
+            
+            In order to add a test to a 'Suite', please use the 'set_params' method.
```

Comment:
In my opinion this message is not very clear. In particular "to add ad test to a Suite, please use set_params", I find it a bit confusing.